worldedit-core/src/main/java/com/boydti/fawe/FaweVersion.java

```diff
@@ -18,7 +18,13 @@ public FaweVersion(int year, int month, int day, int hash, int build) {
 
     public FaweVersion(String version, String commit, String date) {
         String[] split = version.substring(version.indexOf('=') + 1).split("-");
-        this.build = Integer.parseInt(split[1]);
+
+        int build = 0;
+        try {
+            build = Integer.parseInt(split[1]);
+        } catch (NumberFormatException e) {
+        }
+        this.build = build;
```

Comment:
Out of scope.

Reply:
Would you like me to remove it?  I got this error when I installed a development build to my server.  Seemed to me like something that would always break for development builds since they use hashes instead of numbers.